fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -397,6 +400,14 @@ public void addDateValues(List<IDatabaseObject> group, String prefix) {
                 .setTablespace(fhirTablespace)
                 .addPrivileges(resourceTablePrivileges)
                 .enableAccessControl(this.sessionVariable)
+                .addPostStep(priorVersion -> {
```

Comment:
I think the addPostStep should be backed by an ArrayList (I know we talked about a queue as well.  As long as all the statements are deterministic).

Reply:
addPostStep has been replaced by addMigration, but the comment still applies.
the actual list is instantiated in the VersionedSchemaObject constructor:
```
    public VersionedSchemaObject(String schemaName, String objectName) {
        DataDefinitionUtil.assertValidNames(schemaName, objectName);
        this.schemaName = schemaName;
        this.objectName = objectName;
        this.migrations = new ArrayList<>();
    }
```